### ISSUE TITLE
Discover cool new Discord features + add stage channel

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-planning#346]
@WhyldWanderer Since we got over 500 members, it would be awesome if you could look into the new things that this opens up for us and how we can take advantage of it.

We should totally add a "stage channel" for example... so when we launch the GIVeconomy, we can do a GIVeconomy explainer talk without the limit on participants.

**Comment by WhyldWanderer** [comment]
@laurenluz which section of the Discord makes the most sense for the stage channel? Should I just put it with the rest of the voice channels or should it be in a different section? Also, what should we call it?

**Comment by laurenluz** [comment]
I think it's cool to call in stage and put it w the voice channels... what do you think?

**Comment by WhyldWanderer** [comment]
Sounds good to me! we can always change it if we need to...

The channel has beeen added!

I have looked into other perks now that we have 500 members and it looks like all of them are relating to analytics...
The stage channel is a perk of us having a Community server.

Another perk of being a Community server is that we can have Membership Screening where users need to agree to rules before joining. 
![image](https://user-images.githubusercontent.com/74267818/134715375-05dbb776-e1f2-47a9-87b2-c9aebc0da733.png)
We discussed this in the Comms call and I will add this screen once the [Code of Conduct](https://github.com/Giveth/giveth-planning/issues/342) is complets.
